r2/r2/controllers/front.py

```diff
@@ -853,7 +851,7 @@ def GET_related(self, num, article, after, reverse, count):
         end = int(time_module.mktime((article._date + rel_range).utctimetuple()))
         nsfw = u"nsfw:0" if not article.is_nsfw else u""
         query = u"(and %s timestamp:%s..%s %s)" % (query, start, end, nsfw)
-        q = SearchQuery(query, raw_sort="-text_relevance", faceting={},
+        q = g.search.SearchQuery(query, raw_sort="-text_relevance", faceting={},
```

Comment:
Sorry, lost those lines as a result of yesterday's hard reset. Should be good now.